File: app/src/main/java/com/igorganapolsky/vibratingwatchapp/domain/local/TimersDatabase.java
package com.igorganapolsky.vibratingwatchapp.domain.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import com.igorganapolsky.vibratingwatchapp.domain.local.entity.TimerEntity;

@Database(entities = {TimerEntity.class}, version = 7, exportSchema = false)
public abstract class TimersDatabase extends RoomDatabase {

    public abstract TimersDao timersDao();
}

File: app/src/main/kotlin/com/igorganapolsky/vibratingwatchapp/data/TimersDatabase.kt
package com.igorganapolsky.vibratingwatchapp.data

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [TimerEntity::class], version = 7, exportSchema = false)
abstract class TimersDatabase : RoomDatabase() {

    abstract fun timersDao(): ITimersDao
}
